### DISCUSSION TITLE
Screenshot thread

### DISCUSSION THREAD

**Discussion by Anixx** [valinet/ExplorerPatcher#524] [Show and tell]
Let it be a screenshot thread. For once, I shared this on Reddit and got about 200 upvotes (one topic, more than 150 upvotes, the other, more than 50):

![20eom4cp39081](https://user-images.githubusercontent.com/1271736/145011363-4dbb98df-e52b-482a-a40b-5978db2770dc.png)

**Comment by l0oo** [comment]
![W11_classic](https://user-images.githubusercontent.com/49753891/145090625-d71540d3-8d99-47ac-bc1a-165707d70b19.jpg)

**Reply by Anixx** [reply]
Why does your start button not look pushed?

**Reply by l0oo** [reply]
I guess bad settings in _Open-Shell Menu_  somewhere. ;) I've replaced Start button, but without 'Advanced options' maybe.
Will fix it, but TBH do not care too much - I am using TC mostly (also for apps shortcuts, etc).

BTW: My bigger problem is strange rename behaviour in W11 in Explorer: it highlights whole word and jump cursor to begin of name instead of being just where I put the cursor (without highlighting, of course). It is really annoying. Is it typical or I have smthng wrong there? Maybe it is W10 feature?

**Reply by Anixx** [reply]
You do not need to replace it, simply set it to be "Classic", and assign whatever icon and text label you want.

**Reply by Anixx** [reply]
Tried renaming. It highlights the name without the extension part and puts cursor after the highlight (before the dot of if there is no dot, at the end).
This is after enabling SysListView32 in Explorer Patcher.

**Reply by l0oo** [reply]
_This is after enabling SysListView32 in Explorer Patcher._
_highlights the name without the extension part and puts cursor after the highlight_ 

Yup - I guess it is just as you described.
Could it be fixed somehow? :/ To work just like in Windows 7, without helping me.
FYI: I usually change just part of the name of the file (to save a new version of project etc), and it is very annoying. :(

**Reply by Anixx** [reply]
Hmm, if after the name is highlighted you click one more time on the part of the name you want the cursor put at, the cursor will go there and the highlight will disappear. I honestly do not remember how it worked in Windows 7.

**Reply by l0oo** [reply]
Yup - it needs one more click and as many other things in W10/W11 it is just less ergonomic - but why? :( 
Hope there are somewhere options to set: to not highlight, not move cursor - just stay where it was putted.

**Reply by valinet** [reply]
Don’t tell me you expect them to actually improve stuff…?

**Reply by kocane** [reply]
Looks pretty cool. It can't make the taskbar buttons look like the old ones?

**Reply by l0oo** [reply]
Unf. not - it is actually W10 taskbar, which is about 100x better than W11. Classic was the best, but someone decided to remove it. :|
For Windows 95, 98, Me, 2000 look'n'feel you can try RetroBar: https://github.com/dremin/RetroBar - it is limited in functionality but have old school 3D effects.

**Comment by Anixx** [comment]
![изображение](https://user-images.githubusercontent.com/1271736/146625993-b90fad76-f1fa-4ddc-a35f-b83615c7afcc.png)

**Comment by l0oo** [comment]
Looks perfect! Well done! :)
BTW - I fixed (now I see on screenshot above it was 3 parts, shine animated win logo, without push effect) start button acc. to your advice in 'Classic Shell: Forum', now it is taken from W7 winver.exe. You also mention a trick to using gwx.exe to use modern white Win logo. And it works great. Maybe you know where is hidden present W11 modern taskbar blue win logo. Just wanna try it for a moment.
BTW: How did you change network icon in tray.

**Reply by Anixx** [reply]
> How did you change network icon in tray.

This is Network Indicator utility, not the network icon.

Windows logos in System32 directory:
bootux.dll,6 : white, Win10 style

imageres.dll, 250 : white, straight

mstsc.dll, 19 : dark blue, Win10-style

slui.exe, 2 : 4-colored, Win7 style

The last one:
![изображение](https://user-images.githubusercontent.com/1271736/146642687-c7ab38c8-05f2-475c-be10-935f115f7a9c.png)

**Comment by l0oo** [comment]
Thanks for detailed answer!
Mine is now from W7 winver.exe:
![2021-12-18_16-25-06](https://user-images.githubusercontent.com/49753891/146646354-51dccd1f-a0f1-4277-8c65-c468579a2da1.png)

But looking for this, W11 style, it must be somewhere in system... ;) Or it is made diff. way now. :/
![w11](https://user-images.githubusercontent.com/49753891/146646405-399bb3a2-99f0-43fe-b686-61354270e9f7.png)
